projects/plugins/jetpack-boost/dangerfile.js

```diff
@@ -0,0 +1,17 @@
+import { danger, warn } from 'danger';
```

Comment:
Shouldn't be needed anymore.

Reply:
Removed https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025